build_tools/benchmarks/comparisons/simple_commands.py

```diff
@@ -114,10 +114,19 @@ def _generate_cpu(self, device: str):
                            driver="cpu")
     commands.append(tflite)
 
+    tflite_noxnn = TfliteWrapper(self._tflite_benchmark_binary_path,
+                                 self._model_name + "_noxnn",
```

Comment:
I think (not blocking here) we should add a "features" column too: making the features part of the model name makes sorting the final results and comparing across features more cumbersome. Creating pivot tables requires post-processing while with separate features it would be direct. Especially as some of these models have names that are confusing.

Reply:
+1 on this. We do eventually want to include this functionality in the new benchmark infra that Che-Yu is working on. For now, since this code is expected to be replaced, let's aim for doing the minimum.